.travis.yml

```diff
@@ -30,4 +30,6 @@ script:
   - if [[ "$TRAVIS_OS_NAME" == "linux" ]]; then ninja -C build test -v; fi
   - if [[ "$TRAVIS_OS_NAME" == "osx" ]]; then SDKROOT=$(xcodebuild -version -sdk macosx Path) meson build && ninja -C build test; fi
   - dub build
-  - dub test
+  - dub test --build=unittest-cov
+after_success:
+ - bash <(curl -s https://codecov.io/bash)
```

Comment:
There's no real need - sending coverage reports to CodeCov takes about 5-20s and sometimes its good to combine coverage from different models/compilers ...